scripts/build_node_package.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+MODULE_PATH=/tmp/module
+BUILD_TARGET_DIR=/tmp/module_build
+
+git clone https://github.com/mmarchini-oss/node-linux-perf.git $MODULE_PATH
+npm install -g node-gyp
+
+cd $MODULE_PATH
+curl -L https://github.com/nodejs/nan/archive/refs/tags/v2.16.0.tar.gz -o nan.tar.gz
+tar -vxzf nan.tar.gz -C /tmp
+export NAN_PATH=$(realpath /tmp/nan-*)
+sed -i 's/node \-e \\"require('\''nan'\'')\\"/echo $NAN_PATH/g' binding.gyp
```

Comment:
What's this change here?

Reply:
Normally it would be build by npm, which will install `nan` in `.node_modules`, but since I didn't find a way to build it via npm for other versions than npm is installed for, I'm using `node-gyp`. This means I cannot use require (because we don't have `.node_modules`) and I'm providing this module to compiler via path.

Reply:
Understood :) Can you please comment that above the `sed` line?